easycv/resources/download.py

```diff
@@ -0,0 +1,87 @@
+import urllib.error as url_errors
+from urllib.request import urlopen
+
+import shutil
+import hashlib
+
+import easycv.resources.resources as resources
+from easycv.utils import running_on_notebook
+from easycv.errors import ErrorDownloadingResource
+
+# Import the correct tqdm
+if running_on_notebook():
+    from tqdm.notebook import tqdm
+else:
+    from tqdm import tqdm
+
+
+def download_file(filename, url, path, chunk_size=8192, show_progress=False):
+    file_hash = hashlib.sha256()
+    try:
+        response = urlopen(url)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Audit url open for permitted schemes. Allowing use of file:/ or custom schemes is often unexpected.](https://app.codacy.com/gh/easycv/easycv/pullRequest?prid=5625158)